src/services/basketball/news/news.shared.ts

```diff
@@ -0,0 +1,27 @@
+// For more information about this file see https://dove.feathersjs.com/guides/cli/service.shared.html
+import type { Params } from '@feathersjs/feathers'
+import type { ClientApplication } from '../../../client'
+import type { BbNews, BbNewsData, BbNewsPatch, BbNewsQuery, BbNewsService } from './news.class'
+
+export type { BbNews, BbNewsData, BbNewsPatch, BbNewsQuery }
+
+export type BbNewsClientService = Pick<BbNewsService<Params<BbNewsQuery>>, (typeof bbNewsMethods)[number]>
+
+export const bbNewsPath = 'basketball/news'
+
+export const bbNewsMethods = ['find', 'get', 'create', 'patch', 'remove'] as const
```

Comment:
let's remove create patch remove for now

Reply:
Fixed.